docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Plugin/Block/SectionNav.php

```diff
@@ -81,14 +81,15 @@ protected function blockAccess(AccountInterface $account) {
    *   Multidimensional section nav.
    */
   public function getSectionNav() {
+    $renderTree = [];
```

Comment:
This logic change will change the behavior of the following calling code. This should be setting $renderTree to NULL and updating the `@return` statement to match.

```
    $navTree = $this->getSectionNav();
    if ($navTree) {
      $build = [
        '#type' => 'block',
        '#nav_tree' => $navTree,
      ];
    }
    return $build;
```